samples/system-test/microphonestream.test.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2016, Google, Inc.
```

Comment:
Nit: update licenses to be `2018` _( i realize you copy/pasted from an older file, which is what I always do, we just like to use the current date on new files )_